app/views/userProfile.scala.html

```diff
@@ -5,6 +5,7 @@
 @import models.label.LabelTable
 @import models.label.LabelValidationTable
 @(title: String, user: Option[User] = None, auditedDistance: Float)(implicit lang: Lang)
+@userOrg = @{UserOrgTable.getOrg(user.get.userId)}
```

Comment:
You added this up here, but there are still a bunch of places where you could be using it and you aren't! I imagine that you could do the same thing with the output from `OrganizationTable.getOrganizationName()` as well!

And do another pass through to do the refactoring after switching from a `List[Int]` to `Option[Int]`! Some of the methods _work_, but are either redundant or aren't the preferred method:
* `head` -> `get`
* `nonEmpty` -> `isDefined`
* `headOption` is redundant